src/behavioural-events/event.ts

```diff
@@ -47,7 +47,7 @@ export class Event {
       type: this.EVENT_TYPE,
       event_name: this.data.eventName,
       app_user_id: this.data.appUserId,
-      context: buildEventContext(),
+      context: buildEventContext() as unknown as EventProperties,
```

Comment:
Can you move the `as` inside `buildEventContext` and declare it to return `EventProperties`?

In this way we don't need to use `as` in every place we use it.